tests/test_fieldset.py

```diff
@@ -450,9 +450,11 @@ def test_fieldset_write(tmp_zarrfile):
     fieldset.U.to_write = True
 
     def UpdateU(particle, fieldset, time):  # pragma: no cover
+        from parcels._index_search import _search_time_index
+
```

Comment:
Could we move this to the top of the file? (unless there was a specific reason (e..g, circular imports) to put it here?)

Reply:
I wanted to test if an import statement _inside_ a Kernel also worked. But if you strongly prefer to put it at the top then I can do

Reply:
Oh! This is a kernel, of course. ok, that's fine